checks/src/main/java/com/squareup/sqlbrite3/SqlBriteArgCountDetector.kt

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2015 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.sqlbrite3
+
+import com.android.tools.lint.detector.api.Category
+import com.android.tools.lint.detector.api.ConstantEvaluator.evaluateString
+import com.android.tools.lint.detector.api.Detector
+import com.android.tools.lint.detector.api.Implementation
+import com.android.tools.lint.detector.api.Issue
+import com.android.tools.lint.detector.api.JavaContext
+import com.android.tools.lint.detector.api.Scope
+import com.android.tools.lint.detector.api.Severity
+import com.intellij.psi.PsiMethod
+import org.jetbrains.uast.UCallExpression
+
+private const val BRITE_DATABASE = "com.squareup.sqlbrite3.BriteDatabase"
+private const val QUERY_METHOD_NAME = "query"
+private const val CREATE_QUERY_METHOD_NAME = "createQuery"
+
+class SqlBriteArgCountDetector : Detector(), Detector.UastScanner {
+
+  companion object {
+
+    val ISSUE: Issue = Issue.create(
+        "SqlBriteArgCount",
+        "Number of provided arguments doesn't match number " +
+            "of arguments specified in query",
+        "When providing arguments to query you need to provide the same amount of" +
+            "arguments that is specified in query.",
```

Comment:
nit: missing space between "of" and "arguments" inside the concatenation. I would prefer it after "of" similar to the above string.